### PR TITLE
[ASM] - Expander - Expander v2 Pack enhancements - batch 1

### PR DESCRIPTION
**Pull request by andrew-paloalto**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
This update adds a new relatively simple default enrichment playbook for Expander v2 alerts. Additionally it adds several new widgets and a dashboard.

## Screenshots
Screenshot 2023-05-26 at 4 24 31 PM

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by RotemAmit** [comment]
> Thank you for your contribution. Your generosity and caring are unrivaled! Make sure to register your contribution by filling the [Contribution Registration](https://forms.gle/XDfxU4E61ZwEESSMA) form, so our content wizard @RotemAmit will know the proposed changes are ready to be reviewed.

Thank you very much for your contribution! Please register your contribution by filling the [Contribution Registration](https://forms.gle/XDfxU4E61ZwEESSMA) form. Thank you!

File: Packs/CortexXpanse/ReleaseNotes/1_0_4.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+
```
Comment (johnnywilkes) [inline Packs/CortexXpanse/ReleaseNotes/1_0_4.md:1]: Please make sure the release notes (especially prefixes) match the template: https://xsoar.pan.dev/docs/documentation/release-notes#general

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
@andrew-paloalto , did you run `demisto-sdk format` and `demisto-sdk validate` on all the widgets?  That might resolve some of validation errors

File: Packs/CortexXpanse/ReleaseNotes/1_0_4.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+- Added a new widget that highlights the most common IPs involved in Xpanse Incidents.
+
+##### New: Xpanse Incident Count by Business Unit
+
+- Added a new widget that highlights the most common Business Units involved in Xpanse Incidents.
+
+##### New: Unassigned Active Xpanse Incidents
+
+- Added a new widget that includes the number of Active Xpanse Incidents without an assignee.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/ReleaseNotes/1_0_4.md:54]:
```suggestion
Added a new widget that includes the number of active Cortex Xpanse incidents without an assignee.
```

File: Packs/CortexXpanse/ReleaseNotes/1_0_4.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+##### New: Assigned Active Xpanse Incidents
+
+- Added a new widget that includes the number of Active Xpanse Incidents with an assignee.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/ReleaseNotes/1_0_4.md:58]:
```suggestion
Added a new widget that includes the number of active Cortex Xpanse incidents with an assignee.
```

File: Packs/CortexXpanse/ReleaseNotes/1_0_4.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+##### New: Exposed Ports in Xpanse Incidents
+
+- Added a new widget that highlights the most common Xpanse Incidents by port.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/ReleaseNotes/1_0_4.md:62]:
```suggestion
Added a new widget that highlights the most common Cortex Xpanse incidents by port.
```

File: Packs/CortexXpanse/ReleaseNotes/1_0_4.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+##### New: Xpanse Incidents by Protocol
+
+- Added a new widget that highlights the most common Xpanse Incidents by protocol.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/ReleaseNotes/1_0_4.md:66]:
```suggestion
Added a new widget that highlights the most common Cortex Xpanse incidents by protocol.
```

File: Packs/CortexXpanse/ReleaseNotes/1_0_4.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+##### New: New Xpanse High & Critical Incidents
+
+- Added a new widget that includes a timeline view of new high and critical severity Xpanse Incidents.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/ReleaseNotes/1_0_4.md:70]:
```suggestion
Added a new widget that includes a timeline view of new high and critical severity Cortex Xpanse incidents.
```

File: Packs/CortexXpanse/ReleaseNotes/1_0_4.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+#### Layouts
+
+##### Xpanse Alert Layout
+
+- Updated the Xpanse alert layout to include several new fields.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/ReleaseNotes/1_0_4.md:76]:
```suggestion
Updated the Cortex Xpanse alert layout to include several new fields.
```

**Comment by ShirleyDenkberg** [comment]
@johnnywilkes @RotemAmit Doc review completed.

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Enrichment.yml [hunk 8]
```diff
@@ -0,0 +1,334 @@
...
+      root: incident
+      accessor: xpanseip
+  required: false
+  description: ""
+  playbookInputQuery:
+outputs: []
+quiet: true
+tests:
+- No tests (auto formatted)
+fromversion: 6.5.0
```
Comment (johnnywilkes) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Enrichment.yml:334]:
```suggestion
fromversion: 6.8.0
```

File: Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Handler.yml [hunk 9]
```diff
@@ -0,0 +1,266 @@
...
+        "y": -80
+      }
+    }
+  }
+inputs: []
+outputs: []
+quiet: true
+tests:
+- No tests (auto formatted)
+fromversion: 6.5.0
```
Comment (johnnywilkes) [inline Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Handler.yml:266]:
```suggestion
fromversion: 6.8.0
```

**Review by johnnywilkes — COMMENTED** [review]
to fix validation error

File: Packs/CortexXpanse/ReleaseNotes/1_0_4.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
+
+#### Dashboards
+
+##### New: Cortex Xpanse Overview Dashboard
+
+- This dashboard provides a high level view of incidents from Cortex Xpanse.
```
Comment (johnnywilkes) [inline Packs/CortexXpanse/ReleaseNotes/1_0_4.md:6]:
```suggestion
- Added a dashboard that provides a high level view of incidents from Cortex Xpanse.
```

File: Packs/CortexXpanse/ReleaseNotes/1_0_4.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+#### Incident Fields
+
+- New: **Xpanse Protocol**
+- New: **Xpanse Service Validation**
```
Comment (johnnywilkes) [inline Packs/CortexXpanse/ReleaseNotes/1_0_4.md:11]:
```suggestion
- Added the incident field **Xpanse Service Validation** which keeps track of service validation details.
```

File: Packs/CortexXpanse/ReleaseNotes/1_0_4.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+- New: **Xpanse Service ID**
+- New: **Xpanse Provider**
```
Comment (johnnywilkes) [inline Packs/CortexXpanse/ReleaseNotes/1_0_4.md:13]:
```suggestion
- Added the incident field **Xpanse Provider** which keeps track of the provider of the service.
```

File: Packs/CortexXpanse/ReleaseNotes/1_0_4.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+- New: **Xpanse Service Classifications**
```
Comment (johnnywilkes) [inline Packs/CortexXpanse/ReleaseNotes/1_0_4.md:14]:
```suggestion
- Added the incident field **Xpanse Service Classifications** which keeps track of active service classifications.
```

**Review by johnnywilkes — COMMENTED** [review]
I think this should take care of the release note errors

File: Packs/CortexXpanse/IncidentTypes/incidenttype-Xpanse_Alert.json [hunk 14]
```diff
@@ -24,4 +25,4 @@
     "weeks": 0,
     "weeksR": 0,
     "fromVersion": "6.5.0"
```
Comment (johnnywilkes) [inline Packs/CortexXpanse/IncidentTypes/incidenttype-Xpanse_Alert.json:27]:
```suggestion
    "fromVersion": "6.8.0"
```

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
Lets see if this fixed validation errors

**Comment by johnnywilkes** [comment]
@RotemAmit , what do you recommend for the current validation error?  This all stems from me setting `fromversion` for `GCP - Enrichment` playbook in https://github.com/demisto/content/pull/25809/files#diff-79ca7bf9df6cc6a406b7b7bca7c5ab11e17e4250942ad6f1eef54c71a93e380d

`Xpanse_-_Alert_Enrichment` references `GCP - Enrichment`, `Xpanse - Alert Handler` references `Xpanse_-_Alert_Enrichment`, and `incidenttype-Xpanse_Alert.json` references `Xpanse - Alert Handler`

**Comment by RotemAmit** [comment]
> @RotemAmit , what do you recommend for the current validation error? This all stems from me setting `fromversion` for `GCP - Enrichment` playbook in https://github.com/demisto/content/pull/25809/files#diff-79ca7bf9df6cc6a406b7b7bca7c5ab11e17e4250942ad6f1eef54c71a93e380d
> 
> `Xpanse_-_Alert_Enrichment` references `GCP - Enrichment`, `Xpanse - Alert Handler` references `Xpanse_-_Alert_Enrichment`, and `incidenttype-Xpanse_Alert.json` references `Xpanse - Alert Handler`

I see that you changed the fromVersion in the file Packs/CortexXpanse/IncidentTypes/incidenttype-Xpanse_Alert.json from 6.5.0 to 6.8.0. try to change it back to 6.5.0, and see if it helps. Notice that this change will probably require a change in the field fromVersion to 6.5.0 in all the other files as well.

**Comment by efelmandar** [comment]
@andrew-paloalto @johnnywilkes I reviewed the widgets and dashboard and everything looks great! nice work 👍

**Review by johnnywilkes — APPROVED** [review]
@andrew-paloalto , I approve this PR
@RotemAmit , please merge when possible

**Review by RotemAmit — COMMENTED** [review]
Thank you very much for your contribution!
Please resolve the validation errors regarding the release notes.

**Review by RotemAmit — APPROVED** [review]

**Pull request merged**